rafthttp/pipeline.go

```diff
@@ -16,13 +16,13 @@ package rafthttp
 
 import (
 	"bytes"
+	"context"
```

Comment:
@fanminshi we will use context in 3.2 release. we do not need the compatibility anymore.